### PR TITLE
Added segmentCount to callback and message docs

### PR DESCRIPTION
**Pull request by jmulford-bandwidth**
## For the Committer

- [ ] I made sure that the site looks great and functions perfectly live
- [ ] I ran splell check
- [ ] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-apdocs-DX-31.s3-website-us-east-1.amazonaws.com/

File: apiCallbacks/get-mms.md [hunk 1]
```diff
@@ -30,7 +31,8 @@
 	applicationId=a-yr3jpxasdfh5xh5e35saoi&
 	direction=in&
 	media=[{full-url}/giphy-m-asdf.gif, {full-url}/123_1-m-asdf.smil]
-	messageUri=https%3A%2F%2Fapi.catapult.inetwork.com%2Fv1%2Fusers%2Fu-123%2Fmessages%2Fm-asdf
+	messageUri=https%3A%2F%2Fapi.catapult.inetwork.com%2Fv1%2Fusers%2Fu-123%2Fmessages%2Fm-asdf&
+    segmentCount=1
```
Comment (ssodano) [inline apiCallbacks/get-mms.md]: Not sure this should be added here.

File: apiCallbacks/get-sms.md [hunk 2]
```diff
@@ -28,7 +29,8 @@
 	state=received&
 	applicationId=a-yr3jpxasdfh5xh5e35saoi&
 	direction=in&
-	messageUri=https%3A%2F%2Fapi.catapult.inetwork.com%2Fv1%2Fusers%2Fu-123%2Fmessages%2Fm-asdf
+	messageUri=https%3A%2F%2Fapi.catapult.inetwork.com%2Fv1%2Fusers%2Fu-123%2Fmessages%2Fm-asdf&
+    segmentCount=1
```
Comment (ssodano) [inline apiCallbacks/get-sms.md]: Not sure this should be added here.

File: methods/messages/getMessages.md [hunk 3]
```diff
@@ -48,6 +48,7 @@ Message results are paginated based on the `size` parameter. If the number of me
...
 | deliveryCode        | Numeric value of deliver code, see table for values.                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
 | deliveryDescription | Message delivery description for the respective delivery code                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| segmentCount | The number of segments the message was sent as. This value will always be `1` for MMS messages.
```
Comment (ssodano) [inline methods/messages/getMessages.md]: A quirk here is that any messages sent before this change was made will return a segmentCount of 0, we may want to mention that.

File: methods/messages/getMessagesMessageId.md [hunk 4]
```diff
@@ -31,6 +31,7 @@ We only store the message contents for 30 days. Any messages older than 30 days
...
 | deliveryCode        | Numeric value of deliver code, see table for values.                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
 | deliveryDescription | Message delivery description for the respective delivery code                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| segmentCount | The number of segments the message was sent as. This value will always be `1` for MMS messages.
```
Comment (ssodano) [inline methods/messages/getMessagesMessageId.md]: A quirk here is that any messages sent before this change was made will return a segmentCount of 0, we may want to mention that.

File: methods/messages/getMessages.md (hunk 3 above)
Comment (jmulford-bandwidth) [reply methods/messages/getMessages.md]: Added

File: methods/messages/getMessagesMessageId.md (hunk 4 above)
Comment (jmulford-bandwidth) [reply methods/messages/getMessagesMessageId.md]: Added

File: apiCallbacks/get-mms.md (hunk 1 above)
Comment (jmulford-bandwidth) [reply apiCallbacks/get-mms.md]: This is for callbacks that were sent via GET. If this isn't supposed to be there then would the callbacks sent via POST also not have this?

```
  "media"               : ["mediaUri1", "mediaUri2"...],
  "segmentCount":       : integer
}
```

File: apiCallbacks/get-sms.md (hunk 2 above)
Comment (jmulford-bandwidth) [reply apiCallbacks/get-sms.md]: Same question here as for get-mms

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-apdocs-DX-31.s3-website-us-east-1.amazonaws.com/

File: apiCallbacks/get-mms.md (hunk 1 above)
Comment (ssodano) [reply apiCallbacks/get-mms.md]: Right, but the message URI is one of the fields passed via get, so it shouldnt have segmentCount duplicated within it. Just test this with a GET callback and the messageURI was "https://api.catapult.inetwork.com/v1/users/u-ID/messages/m-ID"

Comment (jmulford-bandwidth) [reply apiCallbacks/get-mms.md]: Oh. I see. But the `segmentCount` is not part of the `messageURI`. It's a whole separate field

Comment (jmulford-bandwidth) [reply apiCallbacks/get-mms.md]: It's like `direction` or `applicationId` unless I'm mistaken

Comment (ssodano) [reply apiCallbacks/get-mms.md]: Yes, the segmentCount is not part of the messageURI, the messageURI is a separate piece of data returned in the callback.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-apdocs-DX-31.s3-website-us-east-1.amazonaws.com/

File: apiCallbacks/sms.md [hunk 5]
```diff
@@ -77,7 +78,8 @@ Bandwidth API sends this event to the application when an SMS is sent or receive
   "state"               : "string",
   "deliveryState"       : "string",
   "deliveryCode"        : "string",
-  "deliveryDescription" : "string"
+  "deliveryDescription" : "string",
+  "segmentCount"        : integer
```
Comment (Aubron) [inline apiCallbacks/sms.md]: Quotation Marks - just so we have valid json

File: apiCallbacks/mms.md [hunk 6]
```diff
@@ -40,7 +41,8 @@ Events sent to your server for inbound and outbound MMS messages.
   "deliveryState"       : "string",
   "deliveryCode"        : "string",
   "deliveryDescription" : "string",
-  "media"               : ["mediaUri1", "mediaUri2"...]
+  "media"               : ["mediaUri1", "mediaUri2"...],
+  "segmentCount":       : integer
```
Comment (Aubron) [inline apiCallbacks/mms.md]: quotation marks for valid JSON

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-apdocs-DX-31.s3-website-us-east-1.amazonaws.com/

**Pull request merged**